### PR TITLE
Fix Issue 17644 - [Downloads] Bad link to changelog for DMD beta version

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

**Comment by jansengb** [comment]
Link for Beta is okay now, but for Nightly there is a link to  changelog/<!--UNDEFINED MACRO:.  Maybe $(NIGHTLIES_DMDV2) should be NIGHTLY...?

File: download.dd [hunk 1]
```diff
@@ -104,7 +104,8 @@ $(DIVC download_channels,
   )
 
   $(DIVC download_channel,
-    $(H2 $(LNAME2 dmd-nightly,DMD Nightly))
+    $(H2 $(LNAME2 dmd-nightly,DMD Nightly ($(NIGHTLIES_DMDV2))))
+    $(LINK2 changelog/$(NIGHTLIES_DMDV2)_pre.html, Changelog)
```
Comment (wilzbach) [inline download.dd]: Should be _pending here

**Comment by wilzbach** [comment]
> Link for Beta is okay now, but for Nightly there is a link to changelog/<!--UNDEFINED MACRO:. 

Thanks, but I can't see this on DAutoTest (click on details at the CI). If you tried this locally, you need to nuke `generated` because this introduces a new "generated" variable.

**Comment by jansengb** [comment]
"nuke generated"? Hmm, CONTRIBUTING.md seems inadequate. See also https://issues.dlang.org/show_bug.cgi?id=17649. 

Anyway the download page generated by DAutoTest looks good, so original bug can be closed. Thanks.

File: download.dd (hunk 1 above)
Comment (CyberShadow) [inline download.dd]: Sorry, so why isn't it?

Comment (wilzbach) [inline download.dd]: Cause there are days where I am traveling and can only do stuff on my phone ;-)

Comment (CyberShadow) [inline download.dd]: Want me to fix that for you then?

Comment (wilzbach) [inline download.dd]: If you think it's critical, feel free to go ahead. Otherwise I can get to it tomorrow.

File: download.dd [hunk 2]
```diff
@@ -83,7 +83,7 @@ $(DIVC download_channels,
   $(BETA
   $(DIVC download_channel,
     $(H2 $(LNAME2 dmd_beta,DMD Beta $(B_DMDV2)-$(B_SUFFIX)))
-    $(LINK2 changelog/$(B_DMDV2).html, Changelog)
+    $(LINK2 changelog/$(B_DMDV2)_pre.html, Changelog)
```
Comment (MartinNowak) [inline download.dd]: So far we made the beta changelog from 2.075.0_pre.dd to 2.075.0.html, something went wrong to make a _pre.html from a (generated?) _pre.dd changelog instead.

**Comment by MartinNowak** [comment]
Ah apparently this was broken with #1817.

**Comment by wilzbach** [comment]
This is superfluous with the new changes to the changelog, e.g.
- https://github.com/dlang/dlang.org/pull/1980
- https://github.com/dlang/dlang.org/pull/1944

**Pull request closed without merging**